src/pages/api/productDateInformation.ts

```diff
@@ -61,9 +56,8 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
             redirectTo(res, '/productDateInformation');
             return;
         }
-
-        let startDate = moment();
-        let endDate = moment().add(100, 'y');
+        let startDate;
```

Comment:
This was added because we need to set the defaults if either of the dates are not entered. For example if we enter start date and no end date it needs to default to 100 years from today. Otherwise no way of defaulting.